CONTRIBUTING.md

```diff
@@ -1,3 +1,19 @@
+## 👨‍💻 Prerequisite Skills to Contribute.
```

Comment:
I think we should remove this dot. It'll look better, IMO.
```suggestion
## 👨‍💻 Prerequisite Skills to Contribute
```

Reply:
how you can say that? tell me base of behind the remove dot.